server/plugin.go

```diff
@@ -73,6 +81,9 @@ func (p *Plugin) OnActivate() error {
 		return err
 	}
 
+	p.logger = p.API
+	//p.client.setLogger(p.API)
```

Comment:
Should we uncomment this? or is it somehow handled somewhere else (and therefore the comment should be removed)?